swift-evolution/_dashboard.html

```diff
@@ -0,0 +1,31 @@
+<section class="evolution-dashboard">
+  <div class="search-bar">
+    <input id="search-filter" class="filter" title="Search proposals" placeholder="Search" type="search" incremental="truez`" />
```

Comment:
Remove the value of the ``incremental="truez`"`` attribute?

```suggestion
    <input id="search-filter" class="filter" title="Search proposals" placeholder="Search" type="search" incremental />
```

Reply:
Thanks! This somehow slipped in.